src/App.tsx

```diff
@@ -8,7 +8,38 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
 export const App: React.FC = () => {
+  const [list, setList] = useState<Todo[]>([]);
+  const [selectedTodoId, setSelectedTodoId] = useState<number>(0);
+  const [todoIsClicked, setTodoIsClicked] = useState(false);
+  const [listModified, setListModified] = useState(list);
+
+  const todoButtonStatus = () => {
+    setTodoIsClicked(false);
+  };
+
+  const listChanging = (changedList: Todo[] | []) => {
+    setListModified(changedList);
+  };
```

Comment:
Please name your functions according to these rules:
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364